tests/test_selection.py

```diff
@@ -134,15 +134,15 @@ def test_binary_2():
 
 def test_simple():
     sp = parse_selection("protein")
-    eq(sp.source, "atom.residue.is_protein\n")
+    eq(sp.source, "atom.residue.is_protein")
```

Comment:
I don't care about newlines, but is this change because of something you changed or just to make the code look simpler?

Reply:
Behavior is slightly different between `astunparse.unparse` and  `ast.unparse`(i.e., latter is not as verbose with parenthesis and linebreaks).